### DISCUSSION TITLE
build error

### DISCUSSION THREAD

**Discussion by Joyce-8** [eclipse-tycho/tycho#3758] [General]
Hello, 
I recently discover about the maven plugin tycho since i maintain an application based on it. 
when i run it via eclipse, it shows this error in the pom.xml of any plugins : Failed to execute mojo org.eclipse.tycho:tycho-compiler-plugin:2.0.0. please what have i to do ?

**Comment by akurtakov** [comment]
In a console do `mvn clean verify` (most likely in the top directory or wherever your top level pom.xml file is). Tycho is a Maven plugin does it's best to see your build on the console.
Also Tycho 2.0.0 is very old (version 4.0.7) is latest so you're strongly advised to update to it if the build is not working for you as version 2.0 is out of support.

**Reply by Joyce-8** [reply]
and how can i update it ?

**Reply by laeubi** [reply]
@Joyce-8 you need to adjust the version reference in the `pom.xml` its a bit hard to give better advice without knowing the project, if it is crucial to your business you can find dedicated support options for Tycho here:
https://github.com/eclipse-tycho/tycho#getting-support
